Error loading tmp directory while running in docker

```
 scanii-cli % docker run ghcr.io/uvasoftware/scanii-cli:latest server
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
panic: stat /tmp: no such file or directory

goroutine 1 [running]:
scanii-cli/cmd/sc/internal/commands.runServer(0xc00009e4e0)
	/home/runner/work/scanii-cli/scanii-cli/cmd/sc/internal/commands/server.go:47 +0xf39
scanii-cli/cmd/sc/internal/commands.ServerCommand.func1(0xc0002cac08?, {0x9e39a3?, 0x4?, 0x9e39a7?})
	/home/runner/work/scanii-cli/scanii-cli/cmd/sc/internal/commands/server.go:125 +0x17
github.com/spf13/cobra.(*Command).execute(0xc0002cac08, {0xebff20, 0x0, 0x0})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:987 +0xab1
github.com/spf13/cobra.(*Command).ExecuteC(0xc0000fa308)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039
main.main()
	/home/runner/work/scanii-cli/scanii-cli/cmd/sc/main.go:84 +0x2da
```

--- a/internal/commands/server/server.go
+++ b/internal/commands/server/server.go
@@ -42,8 +42,12 @@ func RunServer(flags *Flags) {
 	}
 
 	if flags.Data == "" {
+		// Ensure the system temp directory exists — minimal Docker images
+		// (e.g. scratch, distroless) may not include /tmp.
+		if err := os.MkdirAll(os.TempDir(), 0755); err != nil {
+			panic(err)
+		}
 		dir, err := os.MkdirTemp("", "scanii-cli")
-
 		if err != nil {
 			panic(err)
 		}

